src/test/java/org/codejive/properties/TestProperties.java

```diff
@@ -1,8 +1,13 @@
 package org.codejive.properties;
 
-import static org.assertj.core.api.Assertions.*;
-
-import java.io.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.IOException;
```

Comment:
Same

Reply:
Huh? I did the same check before writing that message and it did nothing... let me check what's going on :-)

Reply:
Nevermind, reverted the changes and it works now :)